README.md

```diff
@@ -6,6 +6,8 @@
 
 # ShimmerLayout
 
+🌎  Translations: [🇧🇷](https://github.com/brunomunizaf/ShimmerLayout/blob/master/README_pt-br.md) by [@brunomunizaf](https://twitter.com/brunomuniz_af)
```

Comment:
The link does not work.

Reply:
Also, put your name in the author section as the one who created the translation.